winml/lib/Api.Ort/OnnxruntimeEnvironment.cpp

```diff
@@ -14,12 +14,16 @@ using namespace _winml;
 static bool debug_output_ = false;
 
 static HRESULT GetOnnxruntimeLibrary(HMODULE& module) {
+#if WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_APP)
+  auto out_module = LoadPackagedLibrary(L"onnxruntime.dll", 0);
```

Comment:
Issues when called in a desktop context:
Desktop applications cannot use LoadPackagedLibrary; if a desktop application calls this function it fails with APPMODEL_ERROR_NO_PACKAGE.